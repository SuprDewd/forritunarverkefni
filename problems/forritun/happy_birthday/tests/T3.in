Sveinn
13
kk
