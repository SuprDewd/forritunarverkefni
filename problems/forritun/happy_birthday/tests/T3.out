Hann á afmæli í dag,
hann á afmæli í dag.
Hann á afmæli hann Sveinn,
hann á afmæli í dag.

Hann er 13 ára í dag,
hann er 13 ára í dag.
Hann er 13 ára hann Sveinn,
hann er 13 ára í dag.
